helm/grafana-app/Chart.yaml

```diff
@@ -23,4 +23,4 @@ maintainers:
 engine: gotpl
 annotations:
   application.giantswarm.io/values-schema: https://github.com/giantswarm/grafana-app/raw/master/helm/grafana-app/values.schema.json
-  application.giantswarm.io/readme: https://github.com/giantswarm/grafana-app/raw/master/README.md
+  application.giantswarm.io/readme: https://raw.githubusercontent.com/giantswarm/grafana-app/v[[ .Version ]]/README.md
```

Comment:
@piontec already links to the README here. cc @oponder 

However, it doesn't render in Happa: https://share.getcloudapp.com/2Nu0A0Kz

Why is that?

I changed to the format described in https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/apps_in_happa/

Reply:
I think this is because we haven't taught happa yet how to read this new annotation format. We changed the annotation to bring it in line with the others, and I didn't make the required change in Happa yet.

Reply:
Thanks for finding this and checking that it actually worked! I'm adding a todo to my calendar, not sure if there is an issue for it.. but I'll get it done tmw. It's a quick fix.

Reply:
no issue yet. I'll let you take care. Thanks!

Reply:
can you approve this PR if it's correct? 😂